### PR TITLE
modify title, logo and text

### PR DESCRIPTION
**Pull request by bhaskarhc**
### PR contains below changes :
- Rename title `ci status` --> `OpenEBS CI Pipelines`
- OpenEBS logo replaced with Dark background white logo 
- Home dashboard title changed to `Overview`
- Remove status badge column in Home Dashboard  pipeline table 
- Job name made as unclickable in pipeline-job view 
- All branch pages title updated 

Screenshot 2021-07-05 at 5 40 12 PM

Screenshot 2021-07-05 at 5 40 23 PM

### CODE REVIEW

File: src/app/components/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h5 class="card-title ml-1 m-0 p-0 my-1">OpenEBS CI Pipelines</h5>
             <th scope="col">Created at</th>
             <th scope="col">Release Tag</th>
             <th scope="col">Status</th>
-            <th scope="col">e2e Test</th>
+            <!-- <th scope="col">e2e Test</th> -->
```
Comment (satyapriyamishra222) [inline src/app/components/dashboard/dashboard.component.html:16]: Let's remove commented/unwanted code.

File: src/app/components/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -37,13 +37,13 @@ <h5 class="card-title ml-1 m-0 p-0 my-1">OpenEBS CI Pipelines</h5>
                         routerLink="{{genDetailRouteLink(P.pipeline.id,P.pipeline.project,recent.branch)}}">{{P.pipeline.id}}</a>
                     </td>
                     <td>{{timeConverter(P.pipeline.created_at)}}</td>
-                    <td class="text-center">
-                      <div class="text-size-16">{{P.pipeline.release_tag}}</div>
-                    </td>
                     <td>
+                      <div class="text-size-16 release-tag">{{P.pipeline.release_tag}}</div>
+                    </td>
+                    <!-- <td>
```
Comment (satyapriyamishra222) [inline src/app/components/dashboard/dashboard.component.html:43]: Commented code.

File: src/app/components/dialog/dialog.component.scss [hunk 3]
```diff
@@ -70,8 +70,8 @@
     color: white !important;
   }
   .job-row:hover {
-    .job-name a {
-      color: #007bff !important;
+    .job-name{
+      color: #62b9f3 !important;
```
Comment (satyapriyamishra222) [inline src/app/components/dialog/dialog.component.scss:74]: Let's try to remove !important

File: src/app/components/pipeline-detail/pipeline-detail.component.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { DashboardData } from "../../services/ci-dashboard.service";
 import { ISubscription } from "rxjs/Subscription";
 import { Subscription, Observable, timer, from, pipe } from "rxjs";
-import * as Convert from 'ansi-to-html';
+// import * as Convert from 'ansi-to-html';
```
Comment (satyapriyamishra222) [inline src/app/components/pipeline-detail/pipeline-detail.component.ts:5]: Commented code.

File: src/app/components/pipeline-detail/pipeline-detail.component.ts [hunk 5]
```diff
@@ -77,7 +78,7 @@ export class PipelineDetailComponent implements OnInit {
     D.jobs.forEach(j => {
       if (!this.actJob) {
         this.actJob = j.id
-        console.log("------JobID----", this.actJob);
+        // console.log("------JobID----", this.actJob);
```
Comment (satyapriyamishra222) [inline src/app/components/pipeline-detail/pipeline-detail.component.ts:84]: Pls remove this.

File: src/app/components/pipeline-detail/pipeline-detail.component.ts [hunk 6]
```diff
@@ -94,7 +95,7 @@ export class PipelineDetailComponent implements OnInit {
       obj.push(stageObj);
     })
     this.pipeD = obj
-    console.log("\n\n\n stages \n\n\n ", obj);
+    // console.log("\n\n\n stages \n\n\n ", obj);
```
Comment (satyapriyamishra222) [inline src/app/components/pipeline-detail/pipeline-detail.component.ts:101]: Pls remove this.

File: src/app/components/pipeline-detail/pipeline-detail.component.ts [hunk 7]
```diff
@@ -133,10 +134,12 @@ export class PipelineDetailComponent implements OnInit {
     let data = this.ApiService.getJobLogs(p, b, id)
     // console.log("\t d a t a : : - ", p,b,id, "\n\n\n\n\n ***");
     this.jobLogs = data
-    const c = new Convert();
+    // const c = new Convert(); //Enable to use the component this line and 141*
```
Comment (satyapriyamishra222) [inline src/app/components/pipeline-detail/pipeline-detail.component.ts:140]: Let's get rid of commented code.

File: src/app/components/pipeline-detail/pipeline-detail.component.ts [hunk 8]
```diff
@@ -133,10 +134,12 @@ export class PipelineDetailComponent implements OnInit {
...
+
     //  nst txt = "\u001b[38;5;196mHello\u001b[39m \u001b[48;5;226mWorld\u001b[49m";
     // console.log(c.toHtml(d, { use_classes: true }));
-    this.jobLogs = c.toHtml(d, { use_classes: true })
+
+    // this.jobLogs = c.toHtml(d, { use_classes: true })
```
Comment (satyapriyamishra222) [inline src/app/components/pipeline-detail/pipeline-detail.component.ts:145]: Let's get rid of commented code.

File: src/app/pipelines-dashboard/pipelines-dashboard.component.html [hunk 9]
```diff
@@ -2,7 +2,7 @@
   <div class="rounded border px-2 py-2 bg-white my-3 stop-scroll">
     <div class="row">
       <div class="col-6 my-auto">
-        <h5 class="m-0 p-0">OpenEBS {{Engine}} Pipelines</h5>
+        <h5 class="m-0 p-0">OpenEBS {{getBranchName(Engine)}} Pipelines</h5>
```
Comment (satyapriyamishra222) [inline src/app/pipelines-dashboard/pipelines-dashboard.component.html:5]: Are we passing variables as capital case ?

File: src/app/pipelines-dashboard/pipelines-dashboard.component.ts [hunk 10]
```diff
@@ -24,9 +24,9 @@ export class PipelinesDashboardComponent implements OnInit {
     let url = window.location.pathname.split('/')
     let branch:string = url[2]
     this.Platform = url[3]
-    if (this.Engine == "release-branch"){
-      this.Engine = "ZFS-LocalPV"
-    }
+    // if (this.Engine == "release-branch"){
```
Comment (satyapriyamishra222) [inline src/app/pipelines-dashboard/pipelines-dashboard.component.ts:27]: Let's get rid of commented code.

File: src/app/pipelines-dashboard/pipelines-dashboard.component.html (hunk 9 above)
Comment (bhaskarhc) [reply src/app/pipelines-dashboard/pipelines-dashboard.component.html]: no , variable names will be in camelCase . 
Updated 👍

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**